src/ophyd_async/epics/areadetector/drivers/ad_base.py

```diff
@@ -43,14 +43,14 @@ class DetectorState(str, Enum):
 class ADBase(NDArrayBase):
     def __init__(self, prefix: str, name: str = "") -> None:
         # Define some signals
-        self.acquire = ad_rw(bool, prefix + "Acquire")
-        self.acquire_time = ad_rw(float, prefix + "AcquireTime")
-        self.num_images = ad_rw(int, prefix + "NumImages")
-        self.image_mode = ad_rw(ImageMode, prefix + "ImageMode")
-        self.array_counter = ad_rw(int, prefix + "ArrayCounter")
-        self.array_size_x = ad_r(int, prefix + "ArraySizeX")
-        self.array_size_y = ad_r(int, prefix + "ArraySizeY")
-        self.detector_state = ad_r(DetectorState, prefix + "DetectorState")
+        self.acquire = epics_signal_rw_rbv(bool, prefix + "Acquire")
+        self.acquire_time = epics_signal_rw_rbv(float, prefix + "AcquireTime")
+        self.num_images = epics_signal_rw_rbv(int, prefix + "NumImages")
+        self.image_mode = epics_signal_rw_rbv(ImageMode, prefix + "ImageMode")
+        self.array_counter = epics_signal_rw_rbv(int, prefix + "ArrayCounter")
+        self.array_size_x = epics_signal_r(int, prefix + "ArraySizeX")
+        self.array_size_y = epics_signal_r(int, prefix + "ArraySizeY")
```

Comment:
I notice that `ad_r` adds the `_RBV` suffix, so every replacement of it should do the same:
```suggestion
        self.array_size_y = epics_signal_r(int, prefix + "ArraySizeY_RBV")
```